docs/compile-percona-server.md

```diff
@@ -1,6 +1,8 @@
 # Compile Percona Server for MySQL 8.0 from source
 
-The following instructions install Percona Server for MySQL 8.0.
+<!----ask Eleonora whether we need to add here an instruction for PRO bulds---->
```

Comment:
AFAIK - no, @percona-ysorokin ?

Reply:
Currently there is no difference in building Pro version (apart from specifying a different VERSION_COMMENT parameter)
So, I would also say "no".

Reply:
Thank you. Removing the comment.